lucida/template/cpp/Makefile

```diff
@@ -0,0 +1,22 @@
+all:
+	cd server ; \
+	make all ; \
+	cd ../test; \
+	make all ; \
+	cd ..;
+
+start_server: 
+	cd server; ./template_server 
```

Comment:
I am thinking about that and the register tools will change 'template' into $service_name directly in code. I am not sure if it is a good strategy. Actually when using the tools, keyword 'template' in all the codes will change into $service_name automatically.